includes/class-token-user.php

```diff
@@ -145,32 +164,32 @@ public function get( $key, $hash = true ) {
 		$args    = array(
 			'number'      => 1,
 			'count_total' => false,
+			'fields'      => 'ID',
 			'meta_query'  => array(
 				array(
 					'key'     => $key,
 					'compare' => 'EXISTS',
 				),
 			),
 		);
-		$query   = new WP_User_Query( $args );
-		$results = $query->get_results();
+		$results = get_users( $args );
 		if ( empty( $results ) ) {
 			return false;
 		}
-		$user  = $results[0];
-		$value = get_user_meta( $user->ID, $key, true );
+		$user_id = $results[0];
+		$value   = get_user_meta( $user_id, $key, true );
 		if ( empty( $value ) ) {
 			return false;
 		}
 
 		// If this token has expired destroy the token and return false;
 		if ( isset( $value['expiration'] ) && $this->is_expired( $value['expiration'] ) ) {
-			$this->destroy( $key, $user->ID );
+			$this->destroy( $key, $user_ID );
```

Comment:
The `destroy()` function has no user_id param.

Reply:
Fixed that now.